src/main/java/com/revature/controllers/SellerController.java

```diff
@@ -0,0 +1,37 @@
+package com.revature.controllers;
+
+import com.revature.models.Seller;
+import com.revature.models.ShopProduct;
+import com.revature.services.SellerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.Optional;
+
+@CrossOrigin
+@RestController
+public class SellerController {
+
+    @Autowired
+    SellerService sellerService;
+
+//    @GetMapping("/shop_products/{id}")
```

Comment:
There are no routes available in this function. Should the file be removed completely?